### PR TITLE
add ts

### PR DESCRIPTION
**Pull request by plskz**
hehe

### CODE REVIEW

**Comment by plskz** [comment]
@AvinashDhillor

File: yooo.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log('typescript is love');
```
Comment (AvinashDhillor) [inline yooo.ts:1]: change it to typescript

Comment (AvinashDhillor) [inline yooo.ts:1]: modify this line

File: yooo.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (AvinashDhillor) [inline yooo.ts:2]: add new line Typescript is better

File: yooo.ts [hunk 3]
```diff
@@ -1,2 +1 @@
-console.log('typescript is love');
-
+console.log('typescript ');
```
Comment (plskz) [inline yooo.ts:1]: test

**Review by plskz — COMMENTED** [review]
gdxg

**Pull request merged**